### DISCUSSION TITLE
我在电脑上发现一个奇怪的软件，请问这是什么

### DISCUSSION THREAD

**Discussion by Merlin-cyq** [share121/inter-knot#1719] [其他]
![捕获](https://github.com/user-attachments/assets/8c12e757-2cc7-441d-9f4a-2783146f4f83)

**Comment by justafish8787** [comment]
这是软件

**Comment by KecyKING** [comment]
是地狱(确信）

**Reply by whatcanisaylaoda** [reply]
不要故意贬低。

**Comment by Aventurinefragments** [comment]
前方是地狱（确信）

**Comment by moyudashi66** [comment]
我说前方是天堂你信吗（dog）

**Comment by qingyangprince** [comment]
是12306中国铁路购票宇宙列车特别版

**Comment by hhhhhz222** [comment]
这本来是中国铁路官方购票app铁路12306，被二次元黑客劫持后改成了这样，后来被修复，今天您竟能成功让它重现于世，是不是有高人指点

**Comment by ChevalierDeSangreal** [comment]
这是我老婆，怎么会在你电脑上？

**Comment by blqdswkl** [comment]
罗浮杂俎（确信）

**Comment by 1luik** [comment]
通往崩坏平行宇宙的神秘大门

**Comment by yutong903** [comment]
女神异闻录5皇家版

**Comment by Yitaruose-BaiXu** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >米游社

**Comment by 7Hello12** [comment]
hello

**Reply by 7Hello12** [reply]
hello

**Comment by CocoonedSoul** [comment]
这是电子鸦片，染上了戒不掉的那种。建议把UID发过来，让我给你治疗一下。

**Comment by chen92525456** [comment]
病毒，快卸载了<img src="https://bbs-static.miyoushe.com/static/2024/06/19/deac70547d274ae6ccbcdeed8eb8fd51_8158002364855062023.png" alt="黑塔 来测" width="75" >

**Comment by yivitte** [comment]
首先排除轨子

**Comment by noufing** [comment]
是原神

**Comment by wwwsheepwww** [comment]
在你面前的是仙舟第一剑士你敢跟她对视十秒嘛！

**Comment by lllliufeng** [comment]
无视风险，继续访问！（doge

**Comment by homteam** [comment]
这是宇宙列车购票处

**Comment by weixingstar** [comment]
这是病毒，容易掏空你的钱包<img src="https://bbs-static.miyoushe.com/static/2024/05/10/f3c8cf30743f4db513ff0bb8e97d2192_2166245105613052175.png" alt="开拓者 是你" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/05/10/f3c8cf30743f4db513ff0bb8e97d2192_2166245105613052175.png" alt="开拓者 是你" width="75" >

**Comment by OttoApocalypse01** [comment]
这是一种叫做三月妻的赛博病毒（确信）

**Comment by opqphp** [comment]
二月五<img src="https://img-static.mihoyo.com/communityweb/upload/0a74f8be80231a76381738498edd52a6.png" alt="三月七_尴尬" width="75" >
